helpers/deploymentConfig.ts

```diff
@@ -2509,27 +2509,15 @@ export const globalConfig: NetworkConfig = {
         rewards: [
           {
             asset: "XVS",
-            markets: ["WBTC"],
-            supplySpeeds: ["190258751902587"], // 190258751902587 XVS over 365 days (2628000 blocks)
-            borrowSpeeds: ["190258751902587"], // 190258751902587 XVS over 365 days (2628000 blocks)
+            markets: ["vWETH_Core","vWBTC_Core, vUSDT_Core, vUSDC_Core","vcrvUSD_Core"],
```

Comment:
Is this syntax correct? The second item of the array is the string `"vWBTC_Core, vUSDT_Core, vUSDC_Core"`